package.json

```diff
@@ -65,7 +67,8 @@
     "testPathIgnorePatterns": [
       "/node_modules/",
       "/dist/",
-      "[.]fixtures[.]ts"
+      "[.]fixtures[.]ts",
+      "fixtures[.].*[.]ts"
```

Comment:
As discussed, we should probably use:
```
"^fixtures[.].*[.]ts"
```
and rename existing fixtures to `fixtures.*.ts`.